site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -207,6 +207,22 @@
         dropzone.addEventListener("dragleave", draghandle, false);
         dropzone.addEventListener("drop", dropWithMultipleZips, false);
     </script>
+    <script>
+        $(function() {
+            var paths = Object.keys($.parseJSON('{{ directory_priorities|json_encode|raw }}'));
+            for (let x = 0; x < paths.length; x++){
```

Comment:
It might be better to use `forEach()` here instead of iterating over the array manually.